packages/tpetra/core/example/Lesson02-Map-Vector/lesson02_read_modify_vec.cpp

```diff
@@ -50,6 +50,7 @@
 #include <Tpetra_Core.hpp>
 #include <Tpetra_Vector.hpp>
 #include <Tpetra_Version.hpp>
+#include <Teuchos_CommHelpers.hpp>
```

Comment:
@mhoemmen why is this new header needed? It looks like you only modified typedefs...

Reply:
@lucbv See #3159.  If I removed the `Teuchos_CommHelpers.hpp` header file from `Tpetra_ConfigDefs.hpp`, this file did not compile.  It's brittle for a downstream file to depend on arbitrary includes in an upstream file.